protected/humhub/views/layouts/main.php

```diff
@@ -25,7 +26,7 @@
         <?= $this->render('head') ?>
     </head>
 
-    <body>
+    <body class="<?= implode(' ', DeviceDetectorHelper::getBodyClasses()) ?>">
```

Comment:
Small point: maybe we should return a string in `getBodyClasses()` or create another method.
This would keep the View cleaner?

Reply:
I had the same thought. But I like to keep array, because I think there are less bugs when we want to add/remove classes from the result, e.g. in a custom theme, but mainly in general, I think it's a better practice.

Perhaps later we will want to add an event in this method too.

So 2 options:
- Adding the method `getBodyClassesAsString()`
- Using `<?= Html::beginTag('body', ['class' => DeviceDetectorHelper::getBodyClasses()]) ?>`
- 
What do you prefer?

Reply:
Hmm I kike the second option. What do you think?

Reply:
Yes. I also prefer to avoid too many methods. Done in latest commit.